README.md

```diff
@@ -13,3 +13,17 @@ _Fill this out_
 ## Usage and Examples
 
 _...and this_
+
```

Comment:
oh interesting. You intentionally made a small change in this file to make it easier for PR reviewers to see you specs? #smart